SPEC.md

```diff
@@ -175,10 +177,15 @@ The `containerEdits` field has the following definition:
   * `deviceNodes` (array of objects, OPTIONAL) describes the device nodes that should be mounted:
     * `hostPath` (string, REQUIRED) path of the device on the host.
```

Comment:
Needs updating (hostPath + containerPath)

Reply:
same thing here.
BTW, do you think it makes sense to get rid of hostPath and containerPath in favour of path?

Reply:
Is there any use case for mounting a device at a different path in the container?

Reply:
I think there was a reason. Otherwise hostPath and containerPath would not be used in the current spec. I don't know what is it, however. @RenaudWasTaken should know for sure.

Reply:
With the edits that you are making - "hostPath" and "containerPath" no longer make any sense.

The structure basically describes a deviceNode that must be created within the container. In that definition of the structure the name "path" makes a lot of sense because it will never refer to a path on the host.

Reply:
To clarify I'm in favor of removing `hostPath` and `containerPath` in favor of `path`, in the `deviceNodes` structure, in this PR because this PR changes the definition of that structure, from a simple "mount" instruction to a "create Device" instruction.

It seems reasonable at this point to rename the fields that are going to be changed because of that redifinition.

Reply:
done